salt/states/drbd.py

```diff
@@ -0,0 +1,462 @@
+# -*- coding: utf-8 -*-
+#
+# Author: Nick Wang <nwang@suse.com>
+#
+# Copyright 2019 SUSE LINUX GmbH, Nuernberg, Germany.
```

Comment:
can you remove the copyright? The license should not need to be here either as the apache license does not require it be added to each file.

Reply:
Removed the duplicate license in the new commit.